admin_manual/configuration_server/occ_command.rst

```diff
@@ -108,8 +105,8 @@ Enable an app::
    external enabled
    
 ``app:check-code`` checks if the app uses ownCloud's public API (``OCP``) or 
-private API (``OC_``), and then enables the app. If the app uses the private 
-API it will print a warning::
+private API (``OC_``). If the app uses the private API it will print a
```

Comment:
this was a bug: https://github.com/owncloud/documentation/pull/1098#discussion_r29645920